supervisor/core.py

```diff
@@ -192,6 +192,9 @@ async def setup(self):
         except DockerError:
             self.healthy = False
 
+        # Check free space
+        await self.sys_host.check_free_space()
+
```

Comment:
Let's remove this. We can not do anything and we don't need this information for a startup, because the system was running before.

Reply:
normally we use the load() function on the sys_object to initialize default values.

Reply:
See my comment on the review, we use the load function for such things. I will later also offload the supported check from setup into the right load functions because we have now a clean structure with resolution center.